From 836bcce2ecdd9d6ea0650acc9da4f74239a8463b Mon Sep 17 00:00:00 2001
From: MilhouseVH <milhouseVH.github@nmacleod.com>
Date: Tue, 3 Jul 2018 06:17:27 +0100
Subject: [PATCH] drm/i915: partial revert of
 e9af4ea2b9e7e5d3caa6354be14de06b678ed0fa

The original change leaves the GPU more or less permanently in low power mode
with the GPU running at idle frequency and never boosting which may result in
stutter in the Kodi GUI. Tested on Skylake NUC6i5SYH.
---
 drivers/gpu/drm/i915/i915_request.h | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_request.h b/drivers/gpu/drm/i915/i915_request.h
index eddbd424..3f2895a 100644
--- a/drivers/gpu/drm/i915/i915_request.h
+++ b/drivers/gpu/drm/i915/i915_request.h
@@ -298,14 +298,7 @@ static inline bool i915_request_completed(const struct i915_request *rq)
 
 static inline bool i915_request_started(const struct i915_request *rq)
 {
-	u32 seqno;
-
-	seqno = i915_request_global_seqno(rq);
-	if (!seqno)
-		return false;
-
-	return i915_seqno_passed(intel_engine_get_seqno(rq->engine),
-				 seqno - 1);
+	return false;
 }
 
 static inline bool i915_sched_node_signaled(const struct i915_sched_node *node)
-- 
2.14.1

